magento/magento2
# [#7291] Change the default contact form email template to HTML

Hello,

I don't know if there is a reason for that, but having the contact form email in plain text is not really nice and it makes it difficult to override.

### Description
So just a few changes here:
- move the `submitted_form.html` template from `adminhtml` to `frontend`
- include header and footer in this template
- change the template's declaration into the `email_templates.xml` file to set `type` to `html` and `area` to `frontend`

### Fixed Issues (if relevant)
1. #7291

### Manual testing scenarios
1. Go to the contact form and send a mail
2. Check the mail has the default footer and header

I'm open to discussion on this point if it's done on purpose.

Thanks.
